ditto/metric_computer.py

```diff
@@ -0,0 +1,50 @@
+import os
+import datetime
+import traceback
+import json
+
+import logging
+
+from .store import Store
+from .converter import Converter
+from .metrics.network_analysis import network_analyzer
+
+logger = logging.getLogger(__name__)
+
+class MetricComputer(Converter):
+    '''TODO'''
+
+    def __init__(self, registered_reader_class, input_path, output_format, output_path, by_feeder, verbose=True, **kwargs):
+        '''MetricComputer class CONSTRUCTOR.'''
+        self.by_feeder = by_feeder
+        self.output_format = output_format
+        #Call super
+        Converter.__init__(self, registered_reader_class, None, input_path, output_path, verbose=True, **kwargs)
+
+    def convert(self):
+        '''Cannot call convert.'''
+        raise NotImplementedError("MetricComputer cannot convert. Use Converter instead.")
+
+    def compute(self):
+        '''Compute the metrics'''
+        inputs = self.get_inputs(self.feeder)
+
+        self.configure_reader(inputs)
+        self.reader.parse(self.m)
+
+        self.net = network_analyzer(self.m)
```

Comment:
Is `self.net` a network? Consider renaming to `self.network` for clarity. This can be done as a separate PR.